packages/orbit-components/src/primitives/Primitives.stories.tsx

```diff
@@ -9,247 +10,168 @@ import IllustrationPrimitiveComponent, { SIZE_OPTIONS } from "./IllustrationPrim
 
 const getIcon = source => Icons[source];
 
-export default {
+type MixedPropsAndArgs = React.ComponentProps<typeof BadgePrimitiveComponent> &
+  React.ComponentProps<typeof IllustrationPrimitiveComponent> &
+  React.ComponentProps<typeof ButtonPrimitiveComponent>;
+
+const meta: Meta<MixedPropsAndArgs> = {
   title: "Primitives",
 };
 
-export const BadgePrimitive = ({ icon, className }) => {
-  const Icon = getIcon(icon);
+export default meta;
```

Comment:
For this story file, I haven't specified args (not mutual) as we're presenting three different components.